pricegraph/src/lib.rs

```diff
@@ -7,5 +7,5 @@ mod orderbook;
 #[path = "../data/mod.rs"]
 mod data;
 
-pub use encoding::{TokenId, TokenPair};
+pub use encoding::{Element, TokenId, TokenPair};
```

Comment:
nit: We should probably export `Price` and `Validity` as well as they are needed for constructing `Element`.